lighthouse-cli/test/smokehouse/test-definitions/pwa/pwa2-expectations.js

```diff
@@ -82,6 +77,9 @@ module.exports = [
           scoreDisplayMode: 'manual',
         },
       },
+      runWarnings: {
+        length: 1,
+      },
```

Comment:
This might be a problem. Is it a flakiness issue? The runWarning is "The page loaded too slowly to finish within the time limit. Results may be incomplete."  in [the (current) ToT smoke test](https://github.com/GoogleChrome/lighthouse/pull/11711/checks?check_run_id=1513775529#step:8:221), but there's no warning in the stable smoke test and [so the assertion fails](https://github.com/GoogleChrome/lighthouse/pull/11711/checks?check_run_id=1513775477#step:8:222). Presumably that's not a stable/ToT difference though and the page is just slow?

We could tweak the smoke config timeout, but any ideas why this is happening now?

Reply:
Yeah this is a flakiness issue, it shouldn't be added to expectations.

I'm inclined to ignore this particular warning completely in smokes honestly. I'm not sure it adds any value being notified of it.